twittercloneapp/settings.py

```diff
@@ -13,8 +13,15 @@
 import os
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+TEMPLATE_DIR = os.path.join(BASE_DIR,"Template")
```

Comment:
ここ以外も空白に気を付けてください。`,`後は半角スペースを入れましょう
```suggestion
TEMPLATE_DIR = os.path.join(BASE_DIR, "Template")
```

Reply:
[PEP8のサンプル](https://pep8-ja.readthedocs.io/ja/latest/)を参考にしてみてください。この記事にすべて従う必要はないですが、サンプルに従ってみると標準的なコードになると思います！

Reply:
templateよりもtemplatesの方が良いと思います。